woocommerce/woocommerce-android
# Release Shipping classes & zones

### Description


This PR updates Shared library to version `0.4.0` (points to a commit because of possible issue with releasing tag named version) and makes the Shipping classes & zones feature available for production users.

### Testing instructions


Please install the .apk artifcat from this PR (from the bot comment) and check if you can open list of Shippin classes & zones from settings for both: Jetpack and non Jetpack users (to test both authentication methods).

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/5845095/957dbb98-fe2f-4fba-83d3-9de3fa27614e
